test/TestProjects/MgmtScopeResource/Generated/Extensions/MgmtScopeResourceExtensions.cs

```diff
@@ -346,6 +366,7 @@ public static FakePolicyAssignmentResource GetFakePolicyAssignmentResource(this
         /// </summary>
         /// <param name="client"> The <see cref="ArmClient" /> instance the method will execute against. </param>
         /// <param name="id"> The resource ID of the resource to get. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="client"/> is null. </exception>
```

Comment:
the documents are updated accordingly since the exception might be thrown when this parameter is null.